projects/dummy_PETR/mmdet3d_plugin/datasets/transforms/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+# yapf: disable
+from .transforms_3d import (AddPETR, GlobalRotScaleTransImage,
```

Comment:
Are you sure these funcs are different from the similar funcs in `mmdet3d/datasets/transforms/`

Reply:
I'll consider find some funcs in mmdet3d/datasets/transforms/ for them